shared/gameItems.mjs

```diff
@@ -0,0 +1,53 @@
+import ITEM_TYPES from "./itemTypes.mjs";
+
+export default {
+  bag: {
+    id: "bag",
+    displayName: "bag",
+    imgURL: "bag.png",
+    type: ITEM_TYPES.BACKPACK,
+  },
+  bow: {
+    id: "bow",
+    displayName: "bow",
+    imgURL: "bow.png",
+    type: ITEM_TYPES.WEAPON,
+    weapon: {
+      attack: 15,
+      range: 5,
+    },
+  },
+  "copper ore": {
+    id: "copper ore",
+    displayName: "copper ore",
+    imgURL: "copper-ore.png",
+    type: ITEM_TYPES.RESOURCE,
+    // @TODO: Implement basic crafting #190
+    craft: {
+      sword: () => {},
```

Comment:
Not sure if this is the right place for recipes, are we assuming that it always take a single item to craft another item?

Reply:
I'm not sure either. Probably we'll have recipes including many items.